operator/scheduler/model.go

```diff
@@ -47,21 +47,30 @@ func (s *SchedulerClient) LoadModel(ctx context.Context, model *v1alpha1.Model)
 		Model: md,
 	}
 
-	_, err = grcpClient.LoadModel(ctx, &loadModelRequest, grpc_retry.WithMax(SchedulerConnectMaxRetries))
+	_, err = grcpClient.LoadModel(
+		ctx,
+		&loadModelRequest,
+		grpc_retry.WithMax(SchedulerConnectMaxRetries),
+		grpc_retry.WithBackoff(grpc_retry.BackoffExponential(SchedulerConnectBackoffScalar)),
+	)
 	if err != nil {
 		return err, s.checkErrorRetryable(model.Kind, model.Name, err)
 	}
 
 	return nil, false
 }
 
-func (s *SchedulerClient) UnloadModel(ctx context.Context, model *v1alpha1.Model) (error, bool) {
+func (s *SchedulerClient) UnloadModel(ctx context.Context, model *v1alpha1.Model, conn *grpc.ClientConn) (error, bool) {
 	logger := s.logger.WithName("UnloadModel")
-	conn, err := s.getConnection(model.Namespace)
-	if err != nil {
-		return err, true
+
+	// If the connection is not provided, get a new one
+	var err error
+	if conn == nil {
+		conn, err = s.getConnection(model.Namespace)
+		if err != nil {
+			return err, true
```

Comment:
Just two style nits:
- might be more clear to have a variable of type bool called something like `retryUnload` and return that or to return a type alias to bool just to make it more clear what that represents. 
- also, perhaps having error as the last return value (just to match typical go returns)

Reply:
done in [b62bb19](https://github.com/SeldonIO/seldon-core/pull/5320/commits/b62bb19b1db8fa7d7b1547116997f478e42b2a59) and [f90ea17](https://github.com/SeldonIO/seldon-core/pull/5320/commits/f90ea17e480546deab4165159642f6c871cb1ac6)